### ISSUE TITLE
[feature-request] Add possibility to use toggle, and rotary together in the same `tw` instance

### ISSUE THREAD

**Issue by Pablets** [danpacho/tailwindest#44]
**Is your feature request related to a problem? Please describe.**
I can't use toggle and rotary together in the same `tw` instance,  it would be great inside variants.
Here's an example:

```typescript
const twContainer = tw.variants({
	base: {
          ...
	},
	variants: {
		firstChild: {
			true: {
				marginTop: 'mt-12',
			},
			false: {
				marginTop: 'mt-32',
			},
		},
		lastChild: {
			true: {
				marginBottom: 'mb-12',
			},
			false: {
				marginBottom: 'mb-32',
			},
		},
		anotherProp: {
			a: {
				backgroundColor: 'bg-amber-100',
			},
			b: {
				backgroundColor: 'bg-amber-400',
			},
			c: {
				backgroundColor: 'bg-amber-700',
			},
		},
	},
});

interface ContainerProps extends HTMLAttributes<HTMLDivElement>, GetVariants<typeof twContainer> {
	children: ReactNode;
}

const Container: FC<ContainerProps> = ({
	children,
	firstChild = false,
	lastChild = false,
	anotherProp = 'a',
}): JSX.Element => {
	return (
		<div
			className={twContainer.class({
				firstChild, //Type 'boolean | "true" | "false"' is not assignable to type '"true" | "false"'.
				lastChild: String(lastChild) as 'true' | 'false',
				anotherProp,
			})}
		>
			{children}
		</div>
	);
};
```

**Describe the solution you'd like**
It would be great if you can use both toggle and rotary in the same tw instance. Without that you loss certain flexibility

**Comment by danpacho** [comment]
Thanks! That feature will be included in v2.1 :)

**Comment by danpacho** [comment]
This feature is added with #48 PR! Check it out.😎 And thank you for great idea!
